### ISSUE TITLE
how to override this value?

### ISSUE THREAD

**Issue by swanderz** [dbt-labs/dbt-utils#422]
In previous versions of tsql-utils, we could override the `where` condition defined in some tests of the dbt_utils_integration_tests` project ([source](https://github.com/dbt-labs/dbt-utils/blob/3da9568fa993c9494b48e6c7af276fa545224bdf/integration_tests/models/schema_tests/schema.yml#L70-L75) and included below. This is due to TSQL not supporting boolean datatypes.

in the [definition of sqlserver__test_not_null_where](https://github.com/dbt-msft/tsql-utils/blob/0.8.0/macros/dbt_utils/schema_tests/test_not_null_where.sql#L1-L13) you can see how we'd capture and reassign the `where` kwarg like this:
```sql
  {% set where = kwargs.get('where', kwargs.get('arg')) %}
  {% if where == '_deleted = false' %}
      {% set where = '_deleted = 0' %}
  {% endif %}
```

I believe this is because `dbt_utils.default__test_not_null_where` used to capture a `where` argument but not anymore.

Fortunately, if I understand the `deprecation_warning` correctly, this macro isn't needed anymore, so it's best if I disable the test and point users to the dbt-core functionality with this release. Right @jtcohen6?

https://github.com/dbt-labs/dbt-utils/blob/3da9568fa993c9494b48e6c7af276fa545224bdf/integration_tests/models/schema_tests/schema.yml#L70-L75

**Comment by jtcohen6** [comment]
@swanderz That's right! We're encouraging that users switch to using the built-in `not_null` test, since as of v0.20 they can pass their own custom `where` config.
